stage-0-proposals.md

```diff
@@ -53,3 +59,13 @@ See also the [active proposals](README.md), [stage 1 proposals](stage-1-proposal
 [async-context]: https://github.com/legendecas/proposal-async-context
 [async-context-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2020-07/july-23.md#async-context-updates--for-stage-1
 [string-trim-characters]: https://github.com/Kingwl/proposal-string-trim-characters
+[array-async-from]: https://github.com/js-choi/proposal-array-async-from
+[explicit-this-param]: https://github.com/gilbert/es-explicit-this
+[explicit-this-param-notes]: https://github.com/tc39/notes/blob/master/meetings/2020-02/february-6.md#syntax-for-explicitly-this-argument-for-stage-1
+[this-arg-reflection]: https://github.com/hax/proposal-function-this
+[this-arg-reflection-notes]: https://github.com/tc39/notes/blob/master/meetings/2020-02/february-6.md#function-thisargumentexpected-property
+[index-from-end]: https://github.com/hax/proposal-index-from-end
+[index-from-end-notes]: https://github.com/tc39/notes/blob/master/meetings/2021-01/jan-28.md#index-from-end-syntax
```

Comment:
@hax, are any/all of these proposals still active?

Reply:
Yes, at least they are not inactive. :)  

And I believe it's helpful to list them here, especially those had been presented but not yet advanced to stage 1 proposals,  so others who interest in similar topics could find the precedences and join the work.

Reply:
If they're planned to come back to committee at some point, it's definitely good to list them here.

Reply:
I remember previously we use 🚀 to denote whether the champion want to present it in recent meetings, why not use that?

Reply:
That was never for that, it was for “ready to advance”, and we removed that a long time ago.

Reply:
For stage 0, "ready to advance" seems same as "ready for present". And I found the first commit of this page had it!

Reply:
I am not sure why we remove the "ready" tag. It's useful to know the status, especially now we have tc39/dataset which require more information of proposals.

Reply:
Consensus was that it was often inaccurate, and largely useless since if it’s ready to advance, it would *be on an agenda*.